src/consent-manager-builder/index.tsx

```diff
@@ -176,42 +183,59 @@ export default class ConsentManagerBuilder extends Component<Props, State> {
     }
   }
 
-  initialise = async () => {
+  async componentWillReceiveProps(nextProps) {
```

Comment:
I'm guessing this was added because fetching the list of other destinations is asynchronous? I think it's surprising behavior for only one prop to trigger a re-initialize. Could we just handle this in the wrapping component by not rendering the component until `otherDestinations` is ready?

Reply:
super fair!